build-github-tag

```diff
@@ -4,12 +4,13 @@ import groovy.transform.TupleConstructor
 class TagOnRepo {
     String repo
     String tag
+    boolean latest = false
 }
 
 def tagsToBuild = [
         new TagOnRepo('linagora/james-project', 'openpaas-1.4.0'),
         new TagOnRepo('linagora/james-project', 'openpaas-1.4.1'),
-        new TagOnRepo('linagora/james-project', 'james-project-3.3.0'),
+        new TagOnRepo('linagora/james-project', 'james-project-3.3.0', true),
```

Comment:
please use named parameters because boolean arguments are evil otherwise ( http://docs.groovy-lang.org/latest/html/documentation/#_named_parameters_2 )